src/waveform/renderers/waveformwidgetrenderer.h

```diff
@@ -93,6 +94,14 @@ class WaveformWidgetRenderer {
 
     void setTrack(TrackPointer track);
 
+    double getPlayMarkerPosition() {
+        return m_playMarkerPosition;
+    }
+
+    void setPlayMarkerPosition(double newPos) {
+        m_playMarkerPosition = math_clamp(newPos, 0.0, 1.0);
```

Comment:
Is this actually an debug assert?

Reply:
Oops, there is a [DEBUG_ASSERT in math_clamp](https://github.com/mixxxdj/mixxx/blob/master/src/util/math.h#L36), but it's not checking what I thought it was.